configure.ac

```diff
@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
 AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_SUBST(RPM_VERSION, AC_PACKAGE_VERSION)
```

Comment:
A dash is used by RPM (RHEL/SLES) to separate a version from a release. A release is supposed to apply to the package itself vs its contents (ie only the spec changes, bump the release). If we follow these guidelines then everything (make dist, .srpm building) will just work and no hacking around should be required.